<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginAPP</class>
 <widget class="QMainWindow" name="LoginAPP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginAPP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover:!pressed
{
  	border: 3px solid white;
	background-color: red;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>481</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255, 180);
border-radius: 18px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_TenDangNhap">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>70</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Bahnschrift&quot;;
background-color: rgb(228, 236, 255,10);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#000000;&quot;&gt;Tên đăng nhập:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MaPin">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Bahnschrift&quot;;
background-color: rgb(228, 236, 255,10);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#000000;&quot;&gt;Mã pin: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TenDangNhap">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>157</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 244, 255);
font: 75 13pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MaPin">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>123</y>
       <width>157</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 244, 255);
background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 75 11pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DangNhap">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);</string>
     </property>
     <property name="text">
      <string>Đăng nhập</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ThemTaiKhoan">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);</string>
     </property>
     <property name="text">
      <string>Thêm mới</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 255,0);
color: rgb(254, 88, 91);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#ff5500;&quot;&gt;ĐĂNG NHẬP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/BackGr/bkgr_Login.jpg);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_TrangThai">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Bahnschrift&quot;;
background-color: rgb(222, 227, 255,0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Trạng thái: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_TrangThai</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
